styles.css

```diff
@@ -0,0 +1,129 @@
+body {
+  margin: 0;
+}
+
+.wrapper {
+  margin: auto;
+  width: 80%;
+}
+
+.main-content__title {
+  font-family: 'PT Serif', serif;
+}
+
+.logo-name {
+  font-family: 'PT Serif', serif;
+}
+
+.main-content__description {
+  font-family: 'Roboto', sans-serif;
+}
+
+.menu {
+  font-family: 'Roboto', sans-serif;
+  padding: 47px 0;
+}
+
+.main-content__button {
+  font-family: 'Roboto', sans-serif;
+}
+
+.logo, .menu {
+  display: inline-block;
+}
+
+.logo {
+  position: relative;
+  padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px 'PT Serif', Arial, Helvetica, sans-serif;
+}
+
+.logo__circle {
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  opacity: .7;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
+}
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
+}
+
+.header {
+   display: flex;
+   justify-content: space-between;
+}
+
+.menu__item {
+  position: relative;
+  margin-right: 26px;
+  font: 500 18px/21px 'Roboto', Arial, Helvetica, sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s;
+}
+
+.menu__item:last-child{
+  margin-right: 0;
+}
+
+.menu__item:hover{
+  opacity: .5;
+}
+
+.menu__item:after {
+  position: absolute;
+  display: block;
+  content:'';
+  right: 0;
+  width: 0;
+  height: 3px;
+  background: #0056ec;
+  opacity: 0;
+  transition: opacity .5s, width .5s;
+}
+
+.menu__item:hover:after{
+  opacity: 1;
+  width: 100%;
+}
+
+.active::after {
+  position: absolute;
+  display: block;
+  content:'';
+  left: 0;
+  width: 100%;
+  height: 3px;
+  background: #0056ec;
+  opacity: 1;
+}
+
+.info {
+  display: flex;
+  justify-content: space-between;
+}
+
+.footer {
+  display: flex;
+}
+
+.footer__dots {
+  margin: auto;
+}
+
+.main-content__button {
+   padding: 20px 10px; 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/kastykina_app.creation/pullRequest?prid=6252582)